### PR TITLE
Update event-streaming-integration-guide.md

### PR DESCRIPTION
**Pull request by Brandoncharleskhoo1**
Updated the images and copy as we released a new "Test Event Payload"

# Amplitude Developer Docs PR

## Description

Describe your changes. 

## Deadline

When do these changes need to be live on the site?

## Change type

- [ ] Doc bug fix. Fixes #[insert issue number]. Amplitude contributors include Jira issue number. 
- [ ] Doc update.
- [ ] New documentation.
- [ ] Non-documentation related fix or update.

# PR checklist:

- [ ] My documentation follows the style guidelines of this project.
- [ ] I previewed my documentation on a local server using `mkdocs serve`.
- [ ] Running `mkdocs serve` didn't generate any failures.
- [ ] I have performed a self-review of my own documentation.

@amplitude-dev-docs
@caseyamp

### CODE REVIEW

File: docs/partners/event-streaming-integration-guide.md [hunk 1]
```diff
@@ -135,15 +135,18 @@ This section is a Freemarker template that allows you to construct the payload t
 ## Preview and test
 
 1. Before submitting your configuration for review, test the mock payload that you expect to receive from Amplitude. On the right side of the configuration page, follow these steps to preview and test your configuration. 
-    Configure the test integration instance:
+    Configure the test integration instance and hit **Save**:
```
Comment (caseyamp) [inline docs/partners/event-streaming-integration-guide.md:138]:
```suggestion
    Configure the test integration instance and click **Save**:
```

File: docs/partners/event-streaming-integration-guide.md [hunk 2]
```diff
@@ -135,15 +135,18 @@ This section is a Freemarker template that allows you to construct the payload t
...
 
     !!!note
 
         If you would like to disable *Specify event properties to be sent (optional)*, send an email to integrations@amplitude.com.
+           
+![screenshot of the event test payload page](../assets/images/partners/partner-test-tvent-payload.png)
 
-2. Copy test payload CURL command and paste it into your Command Line Interface (CLI).
+     
+2. Click on the **Test Payload** to generate a CURL command. Copy and paste this CURL command into your Command Line Interface (CLI).
```
Comment (caseyamp) [inline docs/partners/event-streaming-integration-guide.md:149]:
```suggestion
2. Click the **Test Payload** to generate a cURL command. Copy and paste this cURL command into your Command Line Interface (CLI).
```

**Review by caseyamp — COMMENTED** [review]
Just a few changes @Brandoncharleskhoo1

**Comment by caseyamp** [comment]
@Brandoncharleskhoo1 I fixed formatting problems and made a few suggestions. After you commit the suggestions, I'll approve. (Note: I didn't set it to changes requested because of that bug that keeps the Amplify preview from re-building)

**Comment by Brandoncharleskhoo1** [comment]
Thanks for reviewing it Casey! Appreciate it :D

**Review by caseyamp — APPROVED** [review]

**Pull request merged**